src/app/accounts/destiny-account.ts

```diff
@@ -168,9 +164,9 @@ async function generatePlatforms(
           originalPlatformType: destinyAccount.membershipType,
           membershipId: destinyAccount.membershipId,
           platformLabel: PLATFORM_LABELS[destinyAccount.membershipType],
-          destinyVersion: 1,
-          platforms: [destinyAccount.membershipType],
-          lastPlayed: new Date(),
```

Comment:
This was another bug - errored accounts would always show up as the most recently played.